src/mumble/MainWindow.cpp

```diff
@@ -3524,6 +3536,10 @@ QPair<QByteArray, QImage> MainWindow::openImageFile() {
 	return retval;
 }
 
+void MainWindow::logChangeNotPermanent(const QString &actionName, ClientUser * const p) const {
+	g.l->log(Log::Warning, QObject::tr("The '%1' change could not be saved permanently and is lost on restart because %2 does not have a certificate.").arg(actionName).arg(Log::formatClientUser(p, Log::Target)));
```

Comment:
I think 
```suggestion
	g.l->log(Log::Warning, QObject::tr("\"%1\" could not be saved permanently and is lost on restart because %2 does not have a certificate.").arg(actionName).arg(Log::formatClientUser(p, Log::Target)));
```
might result in better log messages when the actual change-strings are inserted.